builtin/pack/builder.go

```diff
@@ -133,39 +133,30 @@ func (b *Builder) Build(
 
 	step.Done()
 
+	var ignore []string
+
+	// We want a copy of the slice, not share it because concurrency within
+	// the runner, etc.
+	ignore = append(ignore, b.config.Ignore...)
+
+	if jobInfo.Local {
+		ignore = []string{"data.db", "data.db.lock"}
```

Comment:
Is it even worth doing this? You can technically change these (unlikely anyone does) and I could see a REALLY weird bug/behavior driving someone angry years from now when this is getting ignored for some reason lol. These files (especiially local) are never big, so just not sure if its worth even doing.

Reply:
Up to you! You did that originally.